addons/mail/models/mail_activity_mixin.py

```diff
@@ -404,6 +404,12 @@ def activity_schedule(self, act_type_xmlid='', date_deadline=None, summary='', n
                 'res_id': record.id,
             }
             create_vals.update(act_values)
+            if (user_field_name := self.env.context.get('mail_activity_server_action_activity_user_field_name')):
```

Comment:
`if ... and not create_vals.get("user_id")` ?

same comment for mail_activity_plan_template.py